plugins/contivconf/contivconf.go

```diff
@@ -108,7 +116,7 @@ type ContivConf struct {
 
 	// configuration loaded from the file
 	config     *config.Config
-	ipamConfig *IPAMConfig // IPAM subnets parsed to net.IPNet
+	ipamConfig *IPAMConfig // IPAM subnets and SRv6 cird setting parsed to net.IPNet
```

Comment:
typo in comment ;)

Reply:
fixed in https://github.com/contiv/vpp/pull/1523/commits/c83df2867da05573e3dc92461eb47cb9306d8774